Makefile

```diff
@@ -5,16 +5,15 @@ BIN_OUTPUT_PATH = bin
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 UNAME_S ?= $(shell uname -s)
 
-.PHONY: build
-build:
-	rm -f $(BIN_OUTPUT_PATH)/lorawan
-	CGO_LDFLAGS="$$CGO_LDFLAGS $(CGO_BUILD_LDFLAGS)" go build $(GO_BUILD_LDFLAGS) -o  $(BIN_OUTPUT_PATH)/lorawan main.go
+lorawan:
+	rm -f lorawan
+	CGO_LDFLAGS="$$CGO_LDFLAGS $(CGO_BUILD_LDFLAGS)" go build $(GO_BUILD_LDFLAGS) -o $@ main.go
 
-module.tar.gz: sx1302 build
+module.tar.gz: lorawan first_run.sh meta.json
 	rm -f $(BIN_OUTPUT_PATH)/module.tar.gz
-	tar czf $(BIN_OUTPUT_PATH)/module.tar.gz $(BIN_OUTPUT_PATH)/lorawan
+	tar czf $(BIN_OUTPUT_PATH)/module.tar.gz $^
```

Comment:
unfamiliar with this ^ format, but I assume it puts the build one level up from bin, since bin was removed from the entry point file path?

Reply:
the `$^` tells it to include all the prereq files into the tar. So now instead of the tar just being the module binary we're bundling the binary, `first_run.sh`, and the `meta.json` into the tar. the `lorawan` binary is now put one level up from bin and the `module.tar.gz` is still put in bin.